setup.md

```diff
@@ -107,11 +107,13 @@ sudo canasta skin enable Vector -i canastaId
 * For more info on finishing up your installation, go to the "After installation" section.
 
 ### After installation
-There's several things you can do to polish up your wiki so it's ready for use:
+Once your wiki is installed, there are many customizations you can do:
 
-* To add popular extensions quickly, visit the [Canasta extensions](#enabling-extensions) page to explore your choices.
-* Add a skin to your wiki by choosing a skin and add a `wfLoadSkin` call to `LocalSettings.php`. For instance, to install Timeless, add: `wfLoadSkin( 'Timeless' );` to `LocalSettings.php`.
-* All `.php` files in the `config/settings/` directory will be loaded as if their contents were in `LocalSettings.php`. In addition, if you want to remove the Canasta footer icon, you can remove the `config/settings/CanastaFooterIcon.php` file.
+* Canasta contains around 10 skins and 150 extensions; it is easy to install any of these by adding the appropriate calls to wfLoadSkin() or wfLoadExtension(). See [Enabling skins](#enabling-skins) and [Enabling extensions](#enabling-extensions) for more information.
+* MediaWiki (and its extensions and skins) can be configured in all sorts of ways, usually within `LocalSettings.php` or similar files. MediaWiki alone has over 1,000 configuration settings; see [here](https://www.mediawiki.org/wiki/Special:MyLanguage/Category:MediaWiki_configuration_settings) for one listing of them.
```

Comment:
It's true that the choice of LocalSettings.php vs. random settings files leaves room for confusion, although the phrase "all sorts of ways" is actually a reference to the thousand-plus configuration settings. I guess that's not obvious?

Reply:
> although the phrase "all sorts of ways" is actually a reference to the thousand-plus configuration settings. I guess that's not obvious?

My point is that it should not be obvious.

I know MediaWiki *has* a bunch of ways to do things--that is the problem when you configure by literally writing executable code--but Canasta is providing a way to deploy MediaWiki and it should point users towards the preferred method rather than providing a lot of choices for the end user to make.

They still have those choices, later, if the provided mechanisms don't do the job, but in the documentation, Canasta should point to one method that the Canasta maintainers want people to use. This will simplify support and reduce the cognitive burden on the end user.

Clear documentation should show *how* to do something, not provide a "choose-your-own adventure".

Reply:
To be clear, documentation for Canasta shouldn't even mention ways to do things that you do not want to support.  It shouldn't say "If you don't do things this way, we can't support you" (because that introduces the concept of choice when your objective is to provide simple, clear directions), it should just say "To get X result, the steps are A, B, C."

Reply:
Sorry, I don't understand. This is all in reference to all the configuration options that MediaWiki provides, like using $wgCapitalLinks if you want to have page names that start with lowercase letters. There's no Canasta or non-Canasta way to allow lowercase letters - you just have to use $wgCapitalLinks. So doesn't it make sense to just link to the whole list of settings?

Reply:
LocalSettings.php is one of the areas where Canasta currently provides significant flexibility to the sysadmin for configuration. This is a major benefit of Canasta for purposes of portability. @hexmode, is there a specific restriction you are proposing for us to place on configuration?